respa_pricing/models/price.py

```diff
@@ -120,6 +121,25 @@ class Meta:
     def __str__(self):
         return self.name
 
+    @property
```

Comment:
Couple points:

1. Assuming not likely to change, would a `cached_property` make more sense to reduce re-running db queries?
2. As we are fetching identical fields from different models, maybe `union()` would be a bit more efficient:

```python
price_types = set(
   price_list.usergroup_prices
   .union(price_list.event_prices.all()
)).values_list("price_type", flat=True).distinct())
```

Reply:
Both are good suggestions. I'll make the changes.

Reply:
Modified to use `cached_property` and `union`. It's much much better now. Thanks for the comment!